sys/dev/ata/ata-lowlevel.c

```diff
@@ -857,9 +859,9 @@ ata_pio_read(struct ata_request *request, int length)
 		/* Process main part of data. */
 		resid = size % 2;
 		if (__predict_false((ch->flags & ATA_USE_16BIT) ||
-		    (size % 4) != 0 || ((uintptr_t)addr % 4) != 0)) {
+		    (size % 4) != 0 || is_aligned(addr, 4) != 0)) {
```

Comment:
Might as well take the use of if_aligned all the way.
```suggestion
		    !is_aligned(size, 4) || !is_aligned(addr, 4))) {
```